languages/csharp/concept-exercises/bitwise-operations/.docs/debug.md

```diff
@@ -0,0 +1,10 @@
+# Debug
+
+When a test fails, a message is displayed describing what went wrong and for which input. You can also use the fact that any [console output][programiz.com-basic-input-output] will be shown too:
```

Comment:
`...shown too. You can write to the console using:`

Reply:
Done!